app/src/app/app.component.html

```diff
@@ -1,2 +1,3 @@
-<h1 class="sr-only" title="{{pageHeading | translate}}">{{ pageHeading | translate }}</h1>
+<h1 class="sr-only" title="{{pageHeading | translate}}"
+    [attr.aria-label]="pageHeading | translate">{{ pageHeading | translate }}</h1>
```

Comment:
does it really need aria-label here? I see it already has "title" and value

Reply:
aria-label removed now from the h1 tag,

Reply:
aria-label removed now from the h1 tag,